lib/srv/sess_test.go

```diff
@@ -63,3 +75,239 @@ func TestParseAccessRequestIDs(t *testing.T) {
 	}
 
 }
+
+var _ Server = (*mockServer)(nil)
```

Comment:
nit: does this assertion is really needed  ?
 
```
registry: &SessionRegistry{
					srv: &mockServer{},
				},
```
The `srv: &mockServer{},` assignment should already checks at compile type if mockServer fulfils the `Server` interface.

Reply:
Removed.